cmd/pctl/main.go

```diff
@@ -84,6 +88,65 @@ func showCmd() *cli.Command {
 	}
 }
 
+func installCmd() *cli.Command {
+	return &cli.Command{
+		Name:      "install",
+		Usage:     "generate resources for installing a profile",
+		UsageText: "pctl --catalog-url <URL> install --subscription-name pctl-profile --namespace default --branch main --config-map-name configmap-name <CATALOG>/<PROFILE>",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "subscription-name",
+				DefaultText: "pctl-profile",
+				Value:       "pctl-profile",
+				Usage:       "The name of the subscription.",
+			},
+			&cli.StringFlag{
+				Name:        "namespace",
+				DefaultText: "default",
+				Value:       "default",
+				Usage:       "The namespace to use for generating resources.",
+			},
+			&cli.StringFlag{
+				Name:        "branch",
+				Value:       "main",
+				DefaultText: "main",
+				Usage:       "The branch to use on the repository in which the profile is.",
+			},
+			&cli.StringFlag{
+				Name:  "config-map-name",
+				Value: "",
+				Usage: "The name of the ConfigMap which contains values for this profile.",
+			},
+		},
+		Action: func(c *cli.Context) error {
+			profilePath, catalogURL, err := parseArgs(c)
+			if err != nil {
+				_ = cli.ShowCommandHelp(c, "install")
+				return err
+			}
+
+			branch := c.String("branch")
+			subName := c.String("subscription-name")
+			namespace := c.String("namespace")
+			configValues := c.String("config-map-name")
```

Comment:
need to check these aren't empty? Or do we get that for free from the CLI package

Reply:
They will have the default value.